Pantheon updates 2025-05-14

bobby285271 opened the pull request:
<!--

pantheon.elementary-gtk-theme: 8.2.0 -> 8.2.1
pantheon.elementary-default-settings: 8.0.3 -> 8.1.0
pantheon.switchboard-plug-keyboard: 8.0.1 -> 8.0.2
pantheon.elementary-dock: 8.0.2 -> 8.1.0
pantheon.gala: 8.1.0 -> 8.2.0

/cc  #406987 #406950

-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

bobby285271 committed:
pantheon.gala: 8.1.0 -> 8.2.2

https://github.com/elementary/gala/compare/8.1.0...8.2.2

bobby285271 committed:
pantheon.elementary-dock: 8.0.2 -> 8.1.1

https://github.com/elementary/dock/compare/8.0.2...8.1.1

bobby285271 committed:
pantheon.switchboard-plug-keyboard: 8.0.1 -> 8.0.2

https://github.com/elementary/switchboard-plug-keyboard/compare/8.0.1...8.0.2

bobby285271 committed:
pantheon.elementary-default-settings: 8.0.3 -> 8.1.0

https://github.com/elementary/default-settings/compare/8.0.3...8.1.0

amz-x reviewed (approved):
Tested changes, approving.

bobby285271 commented:
(We will see how soon [gala 8.2.2](https://github.com/elementary/gala/pulls?q=Release+8.2.2) can happen :joy: )

bobby285271 committed:
pantheon.switchboard-plug-pantheon-shell: 8.1.0 -> 8.2.0

https://github.com/elementary/switchboard-plug-pantheon-shell/compare/8.1.0...8.2.0
